src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LOND_TERM = 7;
+  const LOND_TERM_DISCOUNT = 50;
```

Comment:
There is a typo in the constant name `LOND_TERM`. It should be `LONG_TERM` to correctly represent a long-term rental period. Similarly, `LOND_TERM_DISCOUNT` should be `LONG_TERM_DISCOUNT`. Correct naming is crucial for code readability and maintainability.